src/twisted/python/_setup.py

```diff
@@ -36,12 +36,14 @@
 from setuptools import Extension, find_packages
 from setuptools.command.build_py import build_py
 
-from twisted import copyright
-from twisted.python.compat import _PY3
+# Try not to export anything else in Twisted unless we really have to.
```

Comment:
It'd be nice to see this expanded into general advice about modifying the file under the constraints of `exec` and moved into the docstring.